cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsPlatformResources.java

```diff
@@ -395,4 +403,38 @@ public CloudGateWays gateways(CloudCredential cloudCredential, Region region, Ma
     public CloudIpPools publicIpPool(CloudCredential cloudCredential, Region region, Map<String, String> filters) throws Exception {
         return new CloudIpPools();
     }
+
+    @Override
+    public CloudAccessConfigs accessConfigs(CloudCredential cloudCredential, Region region, Map<String, String> filters) throws Exception {
+        CloudAccessConfigs cloudAccessConfigs = new CloudAccessConfigs();
+        AwsCredentialView awsCredentialView = new AwsCredentialView(cloudCredential);
+        AmazonIdentityManagement client = awsClient.createAmazonIdentityManagement(awsCredentialView);
+        try {
+            ListInstanceProfilesResult listRolesResult = client.listInstanceProfiles();
```

Comment:
We have some logic here, maybe we can try to cover it with junit tests

Reply:
@akanto added tests